sycl/test/invoke_simd/invoke_simd.cpp

```diff
@@ -1,350 +0,0 @@
-// RUN: %clangxx -fsycl -fsyntax-only -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s
```

Comment:
Removal of this test doesn't seem right: it has nothing to do with environment variable you are removing, the test doesn't even launch the binary it builds.

Reply:
While removing SYCL_DEVICE_FILTER, I found esimd_emulator has been deprecated from backend list in SYCL_DEVICE_FILTER (and does not exist in ONEAPI_DEVICE_SELECTOR). I removed esimd_emulator part and test as well. Let me double check the spec first.

Reply:
Please do not remove this test.

Reply:
Thanks all! Evgeniy confirmed that this test is not for emulator-specific and we should keep it. I will fix it.